src/util/mapbox.js

```diff
@@ -403,7 +407,7 @@ export class MapLoadEvent extends TelemetryEvent {
             if (customAccessToken || config.ACCESS_TOKEN) {
                 this.queueRequest({id: mapId, timestamp: Date.now()}, customAccessToken);
             } else {
-                this.errorCb(new Error('A valid Mapbox access token is required to use Mapbox GL JS. To create an account or a new access token, visit https://account.mapbox.com/'));
+                this.errorCb(new Error(AUTH_ERR_MSG));
```

Comment:
Is the message the same as what is stored in AUTH_ERR_MSG?

Reply:
No it isnt, `AUTH_ERR_MSG` is just a code that says "NO_ACCESS_TOKEN". This error is never thrown/logged , its used by the caller(`Map#_authenticate`) to un-authenticate the map in specific cases only.